index.html

```diff
@@ -14,10 +14,11 @@
     <div class="container">
       <form action="">
         <label for="">Alert Me</label>
-        <input type="text" class='form-control' />
+        <input type="text" class='form-control' id="yell-text"/>
         <hr>
-        <input type='submit' value="Yell" class='btn btn-default' />
+        <input type='submit' value="Yell" class='btn btn-default' id="yell-submit" />
```

Comment:
Solid id names. But we might be able to look for other things on the page to isolate vs always use explicit ID's.